docs/rules/no-greek-character.md

```diff
@@ -22,8 +22,9 @@ var str = this.getIntlMessage("i18n.key"); // using an i18n tool instead of regu
 
 ## Rule Options
 
-The rule takes one option, an object, which has one key `includeIdentifier` having boolean values `true` or `false`.
-* In general, we don't check identifiers for l10n/i18n. If you would like to lint them, set `includeIdentifier` as `true` to enforce checking.
+The rule takes one option, an object, which has several keys with boolean values `true` or `false`.
+
+### includeIdentifier

```

Comment:
@banyudu Is the description of `includeIndentifier` for Greek deleted by accident? Could you double-check?